pallets/pallet-governance/src/benchmarking.rs

```diff
@@ -0,0 +1,80 @@
+//! Benchmarking setup for pallet-template
+#![cfg(feature = "runtime-benchmarks")]
+use super::*;
+
+use frame_benchmarking::v2::*;
+use frame_support::assert_ok;
+use frame_system::RawOrigin;
+
+#[benchmarks]
+mod benchmarks {
+	use super::*;
+
+	#[benchmark]
+	fn initiate_proposal() {
+		let caller: T::AccountId = whitelisted_caller();
+		let new_council: T::AccountId = account("council", 0u32, 0u32);
+
+		let call = Box::new(
+			crate::Call::<T>::council_rotate_authorities { new_members: vec![new_council] }.into(),
+		);
+
+		let first_proposal = 1u32;
+		CurrentAuthorities::<T>::set(BTreeSet::from([caller.clone()]));
+
+		#[extrinsic_call]
+		initiate_proposal(RawOrigin::Signed(caller), call);
+
+		// Verify
+		assert!(Proposals::<T>::contains_key(first_proposal));
+	}
+
+	#[benchmark]
+	fn vote() {
+		let caller: T::AccountId = whitelisted_caller();
+		let caller_2: T::AccountId = account("caller_2", 0u32, 0u32);
+
+		let call = Box::new(frame_system::Call::<T>::remark { remark: vec![] }.into());
+		let first_proposal = 1u32;
+
+		CurrentAuthorities::<T>::set(BTreeSet::from([caller.clone(), caller_2.clone()]));
+		let _ = Pallet::<T>::initiate_proposal(RawOrigin::Signed(caller.clone()).into(), call);
```

Comment:
Should always use assert_ok if possible.

Reply:
avoid using `let _ =` if possible. Only use it if we are sure it is OK to throw away the result.